plugins/modules/kutils.py

```diff
@@ -0,0 +1,372 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2024 Alexander Bakanovskii <skottttt228@gmail.com>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = r'''
+---
+module: kutils
+short_description: Kerberos utils for managing tickets
+version_added: 9.5.0
+description:
+  - Manage Kerberos tickets with C(kinit), C(klist) and C(kdestroy) base utilities.
+  - See U(https://web.mit.edu/kerberos/krb5-1.12/doc/user/user_commands/index.html) for reference.
+author: "Alexander Bakanovskii (@abakanovskii)"
+attributes:
+  check_mode:
+    support: full
+  diff_mode:
+    support: none
+options:
+  password:
+    description:
+      - Principal password.
+      - It is required to specify O(password) or O(keytab_path).
+    type: str
+  principal:
+    description:
+      - The principal name.
+      - If not set, the user running this module will be used.
+    type: str
+  state:
+    description:
+      - The state of the Kerberos ticket.
+      - V(present) is equivalent of C(kinit) command.
+      - V(absent) is equivalent of C(kdestroy) command.
+    type: str
+    default: present
+    choices: ["present", "absent"]
+  kdestroy_all:
+    description:
+      - When O(state=absent) destroys all credential caches in collection.
+      - Equivalent of running C(kdestroy -A).
+    type: bool
+  cache_name:
+    description:
+      - Use O(cache_name) as the ticket cache name and location.
+      - If this option is not used, the default cache name and location are used.
+      - The default credentials cache may vary between systems.
+      - If not set the the value of E(KRB5CCNAME) environment variable will be used instead, its value is used to name the default ticket cache.
+    type: str
+  lifetime:
+    description:
+      - Requests a ticket with the lifetime, if the O(lifetime) is not specified, the default ticket lifetime is used.
+      - Specifying a ticket lifetime longer than the maximum ticket lifetime (configured by each site) will not override the configured maximum ticket lifetime.
+      - The value for O(lifetime) must be followed by one of the following delimiters V(s) - seconds, V(m) - minutes, V(h) - hours, V(d) - days.
```

Comment:
The colon here was totally fine, you just need to make sure that the YAML parser doesn't parse this as a one-element dictionary with key `The value for O(lifetime) must be followed by one of the following delimiters` and value `V(s) - seconds, V(m) - minutes, V(h) - hours, V(d) - days.`, by using quotes or things like `>-`:
```suggestion
      - "The value for O(lifetime) must be followed by one of the following delimiters: V(s) - seconds, V(m) - minutes, V(h) - hours, V(d) - days."
```
(Same for the others.)

Reply:
right, thanks :) I'm too used to writing YAML with my IDE showing me common errors but it doesn't work in python